scripts/ci/update_downstream.sh

```diff
@@ -28,13 +28,17 @@ mvn -B versions:set-property versions:commit \
   -Dproperty=dependency.alfresco-community-repo.version \
   "-DnewVersion=${VERSION}"
 
+# Define git identity for commits
+git config user.email "${GIT_EMAIL}"
+git config user.name "${GIT_USERNAME}"
```

Comment:
you created an action for handling this: https://github.com/Alfresco/alfresco-build-tools/blob/master/.github/actions/configure-git-author/action.yml

Reply:
I replaced it with [configure-git-author action](https://github.com/Alfresco/alfresco-build-tools/blob/master/.github/actions/configure-git-author/action.yml).

Reply:
This action represents passing the `--local` option to `git config`. Since the script clones enterprise-repo we are not able to use `configure-git-author` during the script execution.

Reply:
we could add an input to toggle ``--global`` and/or an input to provide a custom path where to run the git config command.

Let me know if you prefer to keep as it is now if you think is not worth it for the current task, so I can open an OPSEXP and address this later

Reply:
I opened this PR https://github.com/Alfresco/alfresco-build-tools/pull/208 to add the `global` option.

Reply:
These lines have been removed from the script. Currently we use `configure-git-author@v1.28.0` with `global` option.